pkg/converter/convert.go

```diff
@@ -168,10 +184,15 @@ func Convert(o Options, pathOptions *clientcmd.PathOptions) error {
 			continue
 		}
 
+		klog.V(5).Infof("context: %q", name)
+
 		//  is it legacy aad auth or is it exec using kubelogin?
 		if !isExecUsingkubelogin(authInfo) && !isLegacyAzureAuth(authInfo) {
```

Comment:
do we want a log in this continue case?

Reply:
nah. the L155 has enough info to tell whether the context is specified. anything not printed here will mean that's continued. besides, i don't think we need to log unmatched context